tests_zemu/tests/basic.spec.js

```diff
@@ -267,14 +272,18 @@ describe("Basic checks", function () {
 
             const scheme = FlowApp.Signature.SECP256K1 | FlowApp.Hash.SHA2_256;
             const path = `m/44'/539'/${scheme}'/0/0`;
-            const resp = await app.getAddressAndPubKey(path);
+            const address = "0123456789abcdef"
```

Comment:
Not any address is a valid address. I noticed `slot_parseSlot` in the app does not check for address validity at the very end of the function. We should add a call to `validateChainAddress` to check `N_slot_store.slot[tmp_slotIdx].address`.

Reply:
For the tests we can use 
 - `0xe467b9dd11fa00df` valid on mainnet, invalid on testnet
 - `0x8c5303eaa26202d6` invalid on mainnet, valid on testnet